components/rule-mgt/org.wso2.carbon.identity.rule.metadata/src/main/java/org/wso2/carbon/identity/rule/metadata/config/FlowConfig.java

```diff
@@ -52,14 +57,14 @@ public static FlowConfig load(File file, FieldDefinitionConfig fieldDefinitionCo
 
         try {
             ObjectMapper mapper = new ObjectMapper();
-            Map<String, List<String>> fieldsDefinitionMapPerFlow = mapper.readValue(file,
+            Map<String, List<Map<String, Object>>> fieldsDefinitionMapPerFlow = mapper.readValue(file,
```

Comment:
Why we need to keep a generic object map as entry ? Can't we have a specific type here